common/src/main/java/com/microsoft/identity/common/internal/ui/webview/AzureActiveDirectoryWebViewClient.java

```diff
@@ -182,7 +189,15 @@ private boolean handleUrl(final WebView view, final String url) {
             } else {
                 Logger.info(methodTag,"This maybe a valid URI, but no special handling for this mentioned URI, hence deferring to WebView for loading.");
                 processInvalidUrl(url);
-                return false;
+
+                // Request headers submitted in the initial loadUrl will no longer be propagated for redirects on API31+.
```

Comment:
Since this is webview version related and not API31+, should we call that out in the comment.

Reply:
Yes, I will change this.